modeller/chapter.md

```diff
@@ -335,21 +346,21 @@ For example, the call list for a Cube draws the 6 faces of the cube, with the ce
 ((-0.5, -0.5, 0.5), (-0.5, -0.5, -0.5), (0.5, -0.5, -0.5), (0.5, -0.5, 0.5)),
 # Top face
 ((-0.5, 0.5, -0.5), (-0.5, 0.5, 0.5), (0.5, 0.5, 0.5), (0.5, 0.5, -0.5))
-````````````````````````````````````````
+``````````````````````````````````````````
 
-Only using primitives would be quite limiting for modelling applications. 3D models are generally made up of multiple primitives
+Using only primitives would be quite limiting for modelling applications. 3D models are generally made up of multiple primitives
 (or triangular meshes, which are outside the scope of this project). 
-Fortunately, our design of the Node class facilitates Scene nodes that are made up of multiple primitives. In fact, we can support arbitrary groupings
-of Nodes with no added complexity.
+Fortunately, our design of the `Node` class facilitates `Scene` nodes that are made up of multiple primitives. In fact, we can support arbitrary groupings
+of nodes with no added complexity.
 
-As motivation, let us consider a very basic figure, such as a typical snow figure, made up of three spheres. Even though the figure is comprised of three separate primitives, we would like to be able to treat it as a single object.
+As motivation, let us consider a very basic figure: a typical snowman, or snow figure, made up of three spheres. Even though the figure is comprised of three separate primitives, we would like to be able to treat it as a single object.
 
-We create a class called HierarchicalNode, a Node that contains other nodes. It manages a list of 'children'.
-The `render_self` function for heirarchical nodes is no more complicated than calling render_self on each of the child nodes.
-With the HierarchicalNode class, it is very easy to add figures to the scene.
-Now, defining the snow figure is as simple as specifying the shapes that comprise it and their relative positions and sizes.
+We create a class called `HierarchicalNode`, a node that contains other nodes. It manages a list of 'children'.
```

Comment:
Fair enough. I changed the first node to `Node` but left the second one as it is, mainly because I don't like plurals of words formatted as code -- like `Node`s -- and I can't figure out how to rewrite the sentence so the second node isn't plural. I think it reads okay like this. 

While I'm at it I also changed the single quotes around children to double-quotes, as that's the style we're using (and I should have caught it before!)